src/main/java/org/triplea/client/ui/javafx/controls/TripleACheckBoxSkin.java

```diff
@@ -0,0 +1,56 @@
+package org.triplea.client.ui.javafx.controls;
+
+import javafx.beans.binding.Bindings;
+import javafx.geometry.Pos;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.SkinBase;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.StackPane;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Rectangle;
+
+public class TripleACheckBoxSkin extends SkinBase<CheckBox> {

```

Comment:
Not a regression because I confirmed current `master` has the same behavior, but it would be nice if the check box supported keyboard manipulation.  Some things I noted:

1. There is no visual indication when the check box has the focus.
1. When the check box has the focus, pressing <kbd>Space</kbd> does not toggle the selection.

Reply:
Thought about that too, unfortunately the CSS doesn't give enough control to edit the visuals that much.
Normally the skin shouldn't define any listeners on the created elements, that should be the job of Behavior classes, but those are still not public API (planned, but not today), so we need to define them in the Skin.

It might be the perfect task for you to get used to JavaFX ^^